cli/pkg/platform/platform_openshift.go

```diff
@@ -152,6 +152,6 @@ func (p openShiftPlatform) authenticateAtCluster() (bool, error) {
 }
 
 func (p openShiftPlatform) printCreds() {
-	fmt.Println("Openshift Server URL: " + p.creds.OpenshiftURL)
-	fmt.Println("Openshift User: " + p.creds.OpenshiftUser)
+	fmt.Println("OpenShift Server URL: " + p.creds.OpenShiftURL)
+	fmt.Println("OpenShift User: " + p.creds.OpenShiftUser)
```

Comment:
While I'm correcting the branding for `OpenShift`, I try to skip field names because reasoning through them is too hard. (I missed this one, I'll push a fix shortly.)